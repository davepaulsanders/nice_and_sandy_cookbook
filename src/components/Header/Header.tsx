const Header = () => {
	return (
		<main className="flex flex-col justify-content-center items-center mt-8">
			<h1 className="text-5xl font-bold text-lgg mb-2">Welcome</h1>
			<h2 className="subtitle text-4xl mb-2 text-lightg">
				to the Nice & Sandy Cookbook!
			</h2>
			<p className="text-md text-medg">
				A digital helper for exhausting menu decisions
			</p>
			<hr />
		</main>
	);
};
export default Header;
