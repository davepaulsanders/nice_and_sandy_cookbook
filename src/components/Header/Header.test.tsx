import Header from "./Header" 
import { expect, test } from 'vitest'
import {render } from '@testing-library/react'
import '@testing-library/dom'

test('<Header /> renders', () => {
	const { container } = render(<Header />)
	expect(container).toMatchSnapshot()
})
