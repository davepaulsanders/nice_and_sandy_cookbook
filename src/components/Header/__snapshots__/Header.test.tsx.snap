// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> renders 1`] = `
<div>
  <main
    class="flex flex-col justify-content-center items-center mt-8"
  >
    <h1
      class="text-5xl font-bold text-lgg mb-2"
    >
      Welcome
    </h1>
    <h2
      class="subtitle text-4xl mb-2 text-lightg"
    >
      to the Nice & Sandy Cookbook!
    </h2>
    <p
      class="text-md text-medg"
    >
      A digital helper for exhausting menu decisions
    </p>
    <hr />
  </main>
</div>
`;
